### ISSUE TITLE
[Question] Search in visible text in the window

### ISSUE THREAD

**Issue by AlexVKO** [philc/vimium#2416]
QUESTION:
Is is possible to use the `/` only for what you are seeing or some sort of option to enable search starting from the visible chunk of page to the bottom?

CASE:
lots of time I'm reading articles, turorials or something, and when want to copy some text,  I do some similar like this:

1- `/` and type for the query
2- `V` for visual mode and select the text.
3- copy and paste on where I want to.

basically, the problem is on the first step... it sometimes push me to the top of the page... I always need to think in the better word that can match in that paragraph...

thanks!

**Comment by smblott-github** [comment]
Yes, @AlexVKO.  This is an issue.

Workarounds...

Use `v` *then* `/` (within visual mode).  The `v` will (probably) take you into caret mode, which establishes a selection, which will be used as the starting point for `/`.

In visual mode, `o` can be used to select the other end of the selection.  That means that you don't *have to* `/` for the start of the thing you're looking to select.  Instead, you can `/` for some pattern which only occurs once, then use movements and `o` to select the thing you want.

**Comment by AlexVKO** [comment]
@smblott-github oh thanks dude!

Since it's already a issue I will close this.
